src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const convertedObject = {};
+
+  const splited = sourceString.split(';');
+
+  splited.forEach((split) => {
+    if (split.trim()) {
+      const [property, value] = split.split(':');
+
+      convertedObject[property.trim()] = value.trim();
+    }
```

Comment:
Let's also improve the code here (Improved forEach logic):

```suggestion
  splited.forEach((split) => {
      const [property, value] = split.split(':').map((part) => part.trim());

      if (property && value) {
        convertedObject[property] = value;
      }
    }
```